src/styles/card.css

```diff
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --white: #fff;
+  --blue: #00acdc;
+  --navy-blue: #060b35;
+}
+
+.card {
+  padding-top: 32px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin-left: 19px;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 16px 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--navy-blue);
+}
+
+.card__product-code {
+  margin-left: 16px;
```

Comment:
Looks strange. You can add general horizontal padding to the card because all sections have the same padding
<img width="282" alt="image" src="https://user-images.githubusercontent.com/99403198/217799204-e36ce878-5c43-413d-afe6-1a82d057179b.png">